src/app/pages/StoryPage/index.jsx

```diff
@@ -233,6 +234,7 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
         aboutTags={aboutTags}
       />
       <ATIAnalytics data={pageData} />
+      <ChartbeatAnalytics data={pageData} />
```

Comment:
Can we follow what other pages have done (using `ArticlePage` as an example here). By using `jest.mock` here:

https://github.com/bbc/simorgh/blob/115ac892008d74e896135d02c8e2e038c678c7b0/src/app/pages/ArticlePage/index.test.jsx#L22-L25

we end up with this in the snapshot.

https://github.com/bbc/simorgh/blob/115ac892008d74e896135d02c8e2e038c678c7b0/src/app/pages/ArticlePage/__snapshots__/index.test.jsx.snap#L6-L8

I think it's important that we capturing some form of Chartbeat implementation in these snapshots, and will also ensure we are keeping consistent with our other page types.

Reply:
Fixed [here](https://github.com/bbc/simorgh/pull/6747/commits/9de378b28674720e7dad5d237f4f90b1685ceece)